web/packages/design/src/ButtonToggle/ButtonToggle.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { KeyboardEvent, useState } from 'react';
```

Comment:
```suggestion

import { KeyboardEvent, useState } from 'react';
```

You should put a gap between the license and the first import so that tooling doesn't mistakenly interpret the license as a comment that applies to the import.